test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,55 @@
+const agent = require('superagent');
+const statusCode = require('http-status-codes');
+const chai = require('chai');
+
+const expect = chai.expect;
+
+describe('First Api Tests', () => {
+  it('Consume GET Service', async () => {
+    const response = await agent.get('https://httpbin.org/ip');
+
+    expect(response.status).to.equal(statusCode.OK);
+    expect(response.body).to.have.property('origin');
```

Comment:
try to have only 1 expect, something like: 
```suggestion
    expect(response).toMatchObject({
      status: statusCode.OK,
      body: expect.objectContaining({
        origin: expect.any(String)
      })
```